### PR TITLE
Add firefox/browsers/best-browser.ftl

### PR DESCRIPTION
**Pull request by alexgibson**
Ref: https://github.com/mozilla/bedrock/pull/9746

### CODE REVIEW

File: en/firefox/browsers/best-browser.ftl [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+best-browser-last-but-not-least = Last but not least, a safe browser should offer tools to help you keep an eye on your accounts. Think of alerts that go straight to your email if any of your accounts get breached or icons that tell you whether a website is encrypted, (i.e., if it’s a good idea to enter your credit number on a shopping site).
+
+# Variables:
+# $monitor (url) - link to https://monitor.firefox.com
+best-browser-firefox-is-offering = { -brand-name-firefox } is offering something new to keep you safe: <a href="{ $monitor }">{ -brand-name-firefox } Monitor</a>. It’s a free service that will alert you if there are any public hacks on your accounts and let you know if your accounts got hacked in the past. Another neat feature is the Green Lock. It looks like a small green icon at the top left side of the browser window. If you’re on { -brand-name-firefox } and see the green lock, it means the website is encrypted and secure. If the lock is grey, you might want to think twice about entering any sensitive information.
```
Comment (peiying2) [inline en/firefox/browsers/best-browser.ftl:33]: `Firefox Monitor` as a brand

File: en-CA/firefox/browsers/best-browser.ftl [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+best-browser-privacy-speed-and-security = Privacy, speed, and security.
+best-browser-how-to-choose-the-best = How to choose the best browser for you.
+best-browser-the-internet-has-become = The internet has become as essential as electricity and running water, so choosing the best browser for you is more important than ever. The internet is a second office, a teacher and sometimes a medical advisor, even if your actual doctor would prefer you didn’t look up your symptoms online.
+best-browser-in-the-mid-nineties = In the mid-nineties, Netscape, { -brand-name-ie } and AOL dominated the landscape. It was a simpler time when the sweet melody of dial-up internet rang across the land. You learned the meaning of patience waiting for web pages to load. Back then, all that mattered was browser speed.
+best-browser-today-is-a-different = Today is a different story. Ads, privacy hacks, security breaches, and fake news might have you looking at other qualities in a browser. How does the browser protect your privacy? Does it allow trackers to follow you across the web? Does it built to multitask and handle many computer and internet operations at once?
```
Comment (peiying2) [inline en-CA/firefox/browsers/best-browser.ftl:14]: Should be `Is it built` not `Does it built` in `Does it built to multitask...?`

File: en-CA/firefox/browsers/best-browser.ftl [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+
+### URL: https://www-dev.allizom.org//firefox/browsers/best-browser/
+
+best-browser-find-your-best-browser = Find your best browser for speed, privacy and security.
+best-browser-so-many-browser-options = So many browser options, but there’s only one that works best for your needs. The best browser for you should offer both speed and privacy protection.
+best-browser-privacy-speed-and-security = Privacy, speed, and security.
+best-browser-how-to-choose-the-best = How to choose the best browser for you.
+best-browser-the-internet-has-become = The internet has become as essential as electricity and running water, so choosing the best browser for you is more important than ever. The internet is a second office, a teacher and sometimes a medical advisor, even if your actual doctor would prefer you didn’t look up your symptoms online.
+best-browser-in-the-mid-nineties = In the mid-nineties, Netscape, { -brand-name-ie } and AOL dominated the landscape. It was a simpler time when the sweet melody of dial-up internet rang across the land. You learned the meaning of patience waiting for web pages to load. Back then, all that mattered was browser speed.
```
Comment (peiying2) [inline en-CA/firefox/browsers/best-browser.ftl:13]: `Netscape` shows up multiple times in the [browser history page](https://www-dev.allizom.org/en-US/firefox/browsers/browser-history/). Not sure if it is worth creating an entry. If not, I am fine with that. AOL doesn't appear to have any other mentions in the pages yet to be migrated.

**Review by peiying2 — CHANGES_REQUESTED** [review]

**Comment by alexgibson** [comment]
@peiying2 updated thanks

**Review by peiying2 — CHANGES_REQUESTED** [review]
Looks good, but we need to add an entry of Netscape to the brands.ftl file.

**Comment by alexgibson** [comment]
Great catch! Added

**Review by peiying2 — APPROVED** [review]

**Pull request merged**